documentation.md

```diff
@@ -0,0 +1,14 @@
+# Documentation
+This guide aims to make it clear what information can and cannot be included in documents and designs that are made public.
```

Comment:
At the moment, some of our policies document specific exceptions to policy, which by their nature are likely to be more specific. We may need to rethink this approach.